woocommerce/woocommerce-gateway-stripe
# feat: add payout schedule information

# Changes proposed in this Pull Request:

![2021-10-07 17 48 17](https://user-images.githubusercontent.com/273592/136472639-a53bdda4-2a77-45f1-88ca-68c61bee7168.gif)

# Testing instructions
- Ensure you have the `_wcstripe_feature_upe_settings` flag enabled
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe&panel=settings
- Schedule is added on hover
